### ISSUE TITLE
test set AP

### ISSUE THREAD

**Issue by yjy4231** [SPengLiang/DID-M3D#22]
I randomly selected 200 data as the val set, and the remaining 7281 as the training set. Other settings have not changed. 
The result is low when I submitted to the KITTI benchmark. What is the reason?
![image](https://user-images.githubusercontent.com/52461074/219015759-1e34e727-ad2a-42d5-a884-29576d529841.png)
[train.log](https://github.com/SPengLiang/DID-M3D/files/10742409/train.log)

**Comment by SPengLiang** [comment]
Sorry for the late reply!  We only trained the Car category when using trainval dataset. In our experiments, this manner obtains better performance on Car on KITTI test set. Additionally, we submitted our results two times on KITTI test set, reporting the better one. We find that results on KITTI test set are not very stable, even though we trained the same model twice. Please note that there is a submission limitation on KITTI (3 times in 30 days). Please feel free to contact me if you have further questions.

**Comment by yjy4231** [comment]
thanks a lot!
I have another question
 [https://github.com/SPengLiang/DID-M3D/issues/10#issuecomment-1279779821](url)
![image](https://user-images.githubusercontent.com/52461074/220311561-04a6f832-342e-4eeb-b22f-817899d48f6b.png)
Did you use thefinal ckpt or build a small validation set (e.g., 100 samples) for selecting the best one?  
If you choose 100 as the val set, can you give me your new 'trainval.txt' and 'val.txt'?

**Comment by SPengLiang** [comment]
For the results on KITTI website, we use the trainval set and the final ckpt.

**Comment by yjy4231** [comment]
Got it, I will try it. thanks!
